src/image-templates/default/template.css

```diff
@@ -56,15 +43,16 @@ font-family: 'Jura', sans-serif;
 .description {
 padding-top: 15px;
 color: black;
-font-size: 50px;
+font-size: 40px;
 font-family: 'Jura', sans-serif;
+width: 920px;
+height: 250px;
+overflow: hidden;
```

Comment:
```suggestion
overflow: ellipsis;
```

Reply:
`ellipsis` isn't actually a valid value of `overflow`. You have to have `overflow: hidden` AND `text-overflow: ellipsis`. However I did try this and it simply doesn't work, basically it only works on single lines - see https://discord.com/channels/992103415163396136/992275353961775145/1048657873405751377

Reply:
Oh my bad for giving invalid CSS. But fair enough, that is interesting. But something we could revisit later then.

Reply:
But yeah lets go ahead and tackle these later, I'll merge this one for now